exercises/e001/results/Frydan/b32gen.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/python3
+
+import argparse
+import random
+
+parser = argparse.ArgumentParser(description='Base32 (RFC 4648) random character string generator')
+meg = parser.add_mutually_exclusive_group()
+parser.add_argument('-c', '--characters', type=int, default=32, metavar='', help='Number of characters to generate (default: 32)')
```

Comment:
this should be an optional argument, in other words, not `cmd.py -c 12` but `cmd.py 12`

Reply:
while at it, see `argparse` documentation for `%(default)s`